### DISCUSSION TITLE
Computing Binding Free Energy Approach

### DISCUSSION THREAD

**Discussion by josephDeluna** [Valdes-Tresanco-MS/gmx_MMPBSA#482] [Q&A]
I've done multiple dynamics simulation of different ligands with the same protein, use "gmx trjconv -center" and computed their respective binding free energies. However as I visualize those protein-ligand complexes, I've noticed that some of them split the protein in two and jumps outside of the box. So I use "gmx trjconv -nojump" to fix those complexes.

My questions are: 

1. Do i need to rerun my binding free energy calculations on those complexes that i fix with  "gmx trjconv -nojump" ? 
2.  Those complexes that are ok with "gmx trjconv -center", do i need to do  "gmx trjconv -nojump" to them and rerun my binding free energy calculations for consistency?

**Comment by Valdes-Tresanco-MS** [comment]
1. Probably yes
2. `-nojump` does not exist, in any case, will be `-pbc nojump`. Well `-center` and `-pbc` have different purposes. The first centers the molecule coordinates to the box, while the second intends to remove the PBC. So, the recommendation is to remove the PBC whether the trajectory is visually well or not, and then if you want, center the molecule or align it with your initial structure
